[wundergroundupdatereceiver] Add "Outdoor" to humidity

danieldemus opened the pull request:
Signed-off-by: Daniel Demus <daniel-github@demus.dk>

# Title
[wundergroundupdatereceiver] Update the Humidity label and description to better differentiate from the indoor one

# Description
In relation to fixing the danish translations an improvement was suggested in https://github.com/openhab/openhab-addons/pull/10105#issuecomment-1160791717.

# Testing
URL TBD
Your pull request will automatically be built and available under the following folder:
https://openhab.jfrog.io/ui/native/libs-pullrequest-local/org/openhab/addons/bundles/

It is a good practice to add a URL to your built JAR in this pull request description,
so it is easier for the community to test your Add-on.
If your pull request contains a new binding, it will likely take some time
before it is reviewed and processed by maintainers.
That said, consider submitting your Add-on in the Marketplace:
https://community.openhab.org/c/marketplace/69

jlaur commented:
@danieldemus - thanks! Now I found the other one we [discussed](https://crowdin.com/translate/openhab-addons/6234/en-da?filter=basic&value=3#118602):
```
channel-type.wundergroundupdatereceiver.solarradiation.description = Solar radiation
```

Here unit is missing. Can you fix that also? :-)

danieldemus commented:
> @danieldemus - thanks! Now I found the other one we [discussed](https://crowdin.com/translate/openhab-addons/6234/en-da?filter=basic&value=3#118602):
> 
> ```
> channel-type.wundergroundupdatereceiver.solarradiation.description = Solar radiation
> ```
> 
> Here unit is missing. Can you fix that also? :-)

Done. I also added a unit to visibility, as it is expressed in nautical miles, and changed the querystring related descriptions. How do you update crowdin with the new strings?

jlaur reviewed (approved):
LGTM

jlaur commented:
@lolodomo - default translations haven't been imported automatically in Crowdin. Can they be imported manually? I don't know how to do this.

lolodomo commented:
> @lolodomo - default translations haven't been imported automatically in Crowdin. Can they be imported manually? I don't know how to do this.

I don't know how to do that too :(
But @wborn did it last weekend for another PR.

wborn commented:
You can update them manually using the "Content" tab: https://crowdin.com/project/openhab-addons/content/files

Then expand the tree and select "Update" on the respective file:

![pic](https://user-images.githubusercontent.com/12213581/174963787-7d30e200-519d-4cad-8eb2-4c9e5327f4c3.png)

A "Keep Translations for Updated String" dialog will then open and then to update all translations just hit "Update File".

jlaur commented:
@danieldemus - now ready in Crowdin.
